core/src/main/scala/cats/data/NonEmptyMap.scala

```diff
@@ -0,0 +1,343 @@
+/*
+ * Copyright (c) 2018 Luka Jacobowitz
```

Comment:
Shall we add a copyright header to all files in a separate PR (I am thinking maybe we can use sbt-header)

Reply:
Sounds like a good idea!

Reply:
Shall we use copyright maintainers like the website or each file copyright goes to individual creators?

Reply:
Copyright maintainers seems best to me, I just copy and pasted this one from my own project :smile: